examples/client_metadata_sd_jwt_vc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "vp_formats_supported": {
+        "vc+sd-jwt": {
+            "sd-jwt-alg": [
+                "ES256",
+                "ES384"
+            ],
+            "kb-jwt-alg": [
```

Comment:
Change identifier per above.

Reply:
as above convention-wise, this is correct